common/frame_session.go

```diff
@@ -721,6 +721,8 @@ func (fs *FrameSession) onPageLifecycle(event *cdppage.EventLifecycleEvent) {
 		fs.manager.frameLifecycleEvent(event.FrameID, LifecycleEventLoad)
 	case "DOMContentLoaded":
 		fs.manager.frameLifecycleEvent(event.FrameID, LifecycleEventDOMContentLoad)
+	case "networkIdle":
```

Comment:
I couldn't work out how to replicate this issue in a test 😞 So there are no unit/integration tests

Reply:
I need to work on a test now 😪